From 0e0506d6bf5ebd93de4f729ce0b3fefbe01a091d Mon Sep 17 00:00:00 2001
Message-Id: <0e0506d6bf5ebd93de4f729ce0b3fefbe01a091d.1636639227.git.leonro@nvidia.com>
From: Leon Romanovsky <leonro@nvidia.com>
Date: Thu, 11 Nov 2021 15:50:08 +0200
Subject: [PATCH] build: Don't install files that we are not interested

Attempt to package SimX on FC35 will generate the following errors.
These files are not relevant for SimX at all, so simply don't install
them,

/usr/bin/strip: Unable to recognise the format of the input file
 `/root/rpmbuild/BUILDROOT/mlx-simx-1-1.fc35.x86_64/opt/simx/share/qemu-kvm/hppa-firmware.img'

/usr/bin/strip: Unable to recognise the format of the input file
 `/root/rpmbuild/BUILDROOT/mlx-simx-1-1.fc35.x86_64/opt/simx/share/qemu-kvm/palcode-clipper'

Signed-off-by: Leon Romanovsky <leonro@nvidia.com>
---
 Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Makefile b/Makefile
index 8a9113e6663e..bd8094a1cf3d 100644
--- a/Makefile
+++ b/Makefile
@@ -843,11 +843,9 @@ bamboo.dtb canyonlands.dtb petalogix-s3adsp1800.dtb petalogix-ml605.dtb \
 multiboot.bin linuxboot.bin linuxboot_dma.bin kvmvapic.bin pvh.bin \
 s390-ccw.img s390-netboot.img \
 slof.bin skiboot.lid \
-palcode-clipper \
 u-boot.e500 u-boot-sam460-20100605.bin \
 qemu_vga.ndrv \
 edk2-licenses.txt \
-hppa-firmware.img \
 opensbi-riscv32-sifive_u-fw_jump.bin opensbi-riscv32-virt-fw_jump.bin \
 opensbi-riscv64-sifive_u-fw_jump.bin opensbi-riscv64-virt-fw_jump.bin
 
-- 
2.33.1

